src/components/ResourceList/ResourceList.tsx

```diff
@@ -53,6 +53,8 @@ export interface Props {
   bulkActions?: BulkActionsProps['actions'];
   /** Collection of IDs for the currently selected items */
   selectedItems?: SelectedItems;
+  /** Display header "Select All" button and checkboxes in front of every list item **/
+  selectable?: boolean;
```

Comment:
I wonder if we can be more clear here?

Renders a `Select All` button at the top of the list and checkboxes in front of each list item. For use when `bulkActions` aren't provided.